addinstanceform.php

```diff
@@ -20,11 +20,14 @@
  * @package   cachestore_redis
  * @copyright 2013 Adam Durana
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @var $CFG  stdClass
  */
 
 defined('MOODLE_INTERNAL') || die();
 
+global $CFG;
 require_once($CFG->dirroot.'/cache/forms.php');
+require_once(__DIR__.'/lib.php');
```

Comment:
Almost the same as above.

It declares `cachestore_redis` class (it is the plugin lib, it is a bad Moodle convention but that is where they declare the cachestore class).

If using any cachestore page, this class has been previously included so no problems.

When included in tests, the following line says that the cachestore class not defined:

https://github.com/catalyst/moodle-cachestore_redis/blob/a8735068773a016390cc9bc9fef90d9004b13af2/addinstanceform.php#L70